nixos/modules/i18n/input-method/kime.nix

```diff
@@ -0,0 +1,52 @@
+{ config, pkgs, lib, generators, ... }:
+with lib;
+let
+  cfg = config.i18n.inputMethod.kime;
+  yamlFormat = pkgs.formats.yaml { };
+in
+{
+  options = {
+    i18n.inputMethod.kime = {
+      daemon-config = mkOption {
+        type = yamlFormat.type;
+        default = {
+          modules = ["Xim" "Wayland" "Indicator"];
+        };
+        example = literalExample ''
+          {
+            modules = ["Xim" "Wayland"];
+          }
+        '';
+        description = ''
+          kime daemon.yaml
```

Comment:
Please be a bit more descriptive.

Reply:
Can description have markdown syntax?

Reply:
Example code should be in example or literalExample.

Reply:
I just update version and many documents